06.bowling_object/lib/frame.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require_relative 'shot'
+
+class Frame
+  attr_reader :shots
+
+  def initialize(marks)
+    @shots = marks.map { |m| Shot.new(m) }
+  end
+
+  def score
+    @shots.sum(&:score)
```

Comment:
```suggestion
    pins.sum
```

でも良さそう

同様にstrike?とspare?もpinsを活用した方がシンプルになりそうです

Reply:
score, strike?, spare? の実装をpinsを参照するように修正しました。